pkg/investigations/investigation/investigation.go

```diff
@@ -72,17 +81,21 @@ type Resources struct {
 	ClusterDeployment *hivev1.ClusterDeployment
 	AwsClient         aws.Client
 	BpClient          backplane.Client
+	RestConfig        *Config
```

Comment:
This is one of the cases where a package alias I think makes sense. We have lots of "config" things here and hard to know what this is referring to as is